Documentation/spec/metadata_dotnet_spec.md

```diff
@@ -189,24 +189,24 @@ Only class contains inheritance, the inheritance is a list of spec id.
 
 #### 4.3 Syntax
 
-The syntax part for type contains special languages declaration, and descriptions of type parameters .
-For delegate, it also contains descriptions of paramters and return type.
+The syntax part for type contains a special language declaration, and descriptions of type parameters .
```

Comment:
Here language actually means programming language. Is the following better?

The syntax part for type contains declaration, and descriptions of type parameters for different languages.